doc/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	add_executable (markdownlinkconverter markdownlinkconverter/markdownlinkconverter.c)
+	find_util (markdownlinkconverter MARKDOWN_LINK_CONVERTER "")
+
```

Comment:
> Can you give arguments to the doxygen filter? If not, it won't work with mingw.
No, there is no possibility to pass arguments to a input filter in doxygen, it will be executed by doxygen with popen. I think there is no need to take care of the arguments because doxygen does it.

> I added some docu for find_util: 9ee141a
"EXE_SYM_LOC ... Note: you must pass this value to COMMAND of add_custom_command"
Why should cmake execute the markdownlinkconverter?